spec/instruction.spec.js

```diff
@@ -231,3 +238,33 @@ describe('User Story 5 Test:', () => {
     })
 })
 
+describe('User Story 6 Test:', () => {
+    let testInstruction;
+
+    describe('Weather error test:', () => {
+        const mockPlane = {
+            isLanded: () => false,
+        }
+        const mockAirport = {
+            planesAtAirport: 1,
+            weatherType: 'stormy',
+            landPlane: () => mockPlane.isLanded(),
+            takeOffPlane: () => mockPlane.isLanded(),
+            //checkWeather() { return this.weatherType },
+            getAirportCapacity() {
+                return 5
+            },
+
+        }
+        beforeEach(() => {
+            testInstruction = new Instruction("testInstruction", mockAirport, mockPlane);
+        })
+        afterEach(() => {
+            testInstruction = undefined;
+        })
+        it('6a. expect landPlane to throw an error', () => {
+            // Assert
+            expect(() => { testInstruction.landPlane() }).toThrowError('Too stormy to land / take-off');
+        })
+    })
+})
```

Comment:
What went well:

- With the use of Jasmine as my testing framework, I was able to advance from simple assertion tests. I was also able to test whether the correct methods were called, and also test my error-handling procedures.